bedrock/urls.py

```diff
@@ -17,25 +18,25 @@
 
 urlpatterns = (
     # Main pages
-    url(r'^foundation/', include('bedrock.foundation.urls')),
-    url(r'^grants/', include('bedrock.grants.urls')),
-    url(r'^about/legal/', include('bedrock.legal.urls')),
-    url(r'^press/', include('bedrock.press.urls')),
-    url(r'^privacy/', include('bedrock.privacy.urls')),
-    url(r'^styleguide/', include('bedrock.styleguide.urls')),
-    url(r'^security/', include('bedrock.security.urls')),
-    url(r'', include('bedrock.firefox.urls')),
-    url(r'', include('bedrock.mozorg.urls')),
-    url(r'', include('bedrock.newsletter.urls')),
-    url(r'^etc/', include('bedrock.etc.urls')),
-    url(r'^exp/', include('bedrock.exp.urls')),
-
-    url(r'^healthz/$', watchman_views.ping, name="watchman.ping"),
-    url(r'^readiness/$', watchman_views.status, name="watchman.status"),
-    url(r'^healthz-cron/$', base_views.cron_health_check),
-    url(r'^csp-violation-capture$', base_views.csp_violation_capture,
+    path('foundation/', include('bedrock.foundation.urls')),
+    path('grants/', include('bedrock.grants.urls')),
+    path('about/legal/', include('bedrock.legal.urls')),
+    path('press/', include('bedrock.press.urls')),
+    path('privacy/', include('bedrock.privacy.urls')),
+    path('styleguide/', include('bedrock.styleguide.urls')),
+    path('security/', include('bedrock.security.urls')),
+    path('', include('bedrock.firefox.urls')),
+    path('', include('bedrock.mozorg.urls')),
+    path('', include('bedrock.newsletter.urls')),
+    path('etc/', include('bedrock.etc.urls')),
+    path('exp/', include('bedrock.exp.urls')),
+
+    path('healthz/', watchman_views.ping, name="watchman.ping"),
+    path('readiness/', watchman_views.status, name="watchman.status"),
+    path('healthz-cron/', base_views.cron_health_check),
+    path('csp-violation-capture', base_views.csp_violation_capture,
         name='csp-violation-capture'),
```

Comment:
Sorry. Flake8 is very picky. The indentation of this line needs to line up the `n` under the `'` character from the line above. Same with line 40 below.